Update azure/docker-login action to docker/login-action

hgarvison opened the pull request:
Update azure/docker-login action to docker/login-action to remove Node.js version warning during pipeline runs.

DomAyre commented:
As part of #131 I moved to using the azure cli to login as it remove the need for registry password here, I guess it also solves the node warning

hgarvison commented:
> As part of #131 I moved to using the azure cli to login as it remove the need for registry password here, I guess it also solves the node warning
@DomAyre there were still some instances of the old registry login, so I updated them to use cli login as you did in #131
